src/vsr/client.zig

```diff
@@ -234,7 +234,7 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
             self.request_queue.push_assume_capacity(.{
                 .user_data = user_data,
                 .callback = callback,
-                .message = message.ref(),
+                .message = message,
```

Comment:
I am debating whether we should add `errdefer self.release` for readability and for simmetry with register. I think we'd better not.

Reply:
I prefer the pattern `errdefer self.release()` only after `client.get_message()`.